Library/Formula/mono.rb

```diff
@@ -75,6 +84,17 @@ def install
     EOS
     system "#{bin}/xbuild", xbuild
     assert $?.success?
+
+    # Tests that fsharpi is alive and kicking
+    fsx = testpath/'helloworld.fsx'
+    fsx.write <<-EOS.undent
+    #!/usr/bin/env fsharpi
+    printfn "#{test_str}"
+    EOS
+    system 'chmod +x helloworld.fsx'
+    output = `./helloworld.fsx`
```

Comment:
@mikemcquaid Sorry to correct you. But there is no `system_ouput` in homebrew. It's `shell_output`.